packages/react/src/vertex3.tsx

```diff
@@ -59,7 +59,10 @@ export const Vertex3: React.FunctionComponent<IVertex3> = ({
     const annoOffsetY = 0;
 
     const labelWidth = Utility.textSize(text, textFontFamily, textHeight, false).width;
-    const labelShapeWidth = labelWidth + (textPadding * 2) + (textboxStrokeWidth * 2);
+    let labelShapeWidth = 0;
+    if(text !== ""){
```

Comment:
Can text be `undefined`?

Reply:
I don't think it can because the default value is set to empty string. If someone explicitly set text to `undefined` it should just default to empty string, but I'll test it.

Reply:
Yes, if someone sets text to `undefined` it just defaults to that empty string.